### PR TITLE
Shopping list design.

### LINKED ISSUE #17: 15. As a user, I want the shopping list screen to look pretty
AC:

* The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

shopping-list

### PR DESCRIPTION
**Pull request by skillitzimberg**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
| ✓   | :sunglasses: Styling       |
|  ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Related Issue

closes #17 

## Description

We used eggs to denote purchased state: whole egg - not purchased; cracked egg - purchased.
The color of the buttons the eggs are in are colored according to their next purchase estimate. Blue - soon; yolk yellow - kind of soon; red - not soon; gray - inactive.

The trash/delete item buttons do a nice, subtle switch from a medium gray to dark gray with box shadow on hover.

We used a right angle svg from Font Awesome to open the item detail view/modal.

We installed [React Font Awesome](https://fontawesome.com/how-to-use/on-the-web/using-with/react) because we were having trouble getting the right angle to change to the color we wanted from our theme. React Font Awesome made this really easy.

### Before
Screen Shot 2020-05-26 at 6 44 56 AM

### After
Screen Shot 2020-05-28 at 12 34 13 PM

## Acceptance Criteria

As a user, I want the shopping list screen to look pretty

- The list is pleasing to the eye
- The list easy to understand
- The list is accessible

## Testing Steps / QA Criteria

View the deployed branch [here](https://deploy-preview-50--tcl-6-smart-shopping-list.netlify.app/)
Break stuff. Let us know.

### CODE REVIEW

**Comment by laurenmbeatty** [comment]
Oh my gosh the little eggs are killing me. So great!

**Comment by laurenmbeatty** [comment]
This is gorgeous. I did find a tiny bug. If I have filtered the list, and then delete an item, the entire list shows up again after I delete, rather than staying filtered.

**Comment by skillitzimberg** [comment]
> This is gorgeous. I did find a tiny bug. If I have filtered the list, and then delete an item, the entire list shows up again after I delete, rather than staying filtered.

I think I fixed this.

File: src/components/ListItem.js [hunk 1]
```diff
@@ -59,26 +67,57 @@ const ListItem = ({ item, onDelete, token }) => {
...
+        />
       </button>
-      <Link to={`detail/${item.name_normalized}`}>View Details</Link>
+
+      <Link to={`detail/${item.name_normalized}`}>
+        <FontAwesomeIcon
+          icon={faAngleRight}
+          size="3x"
+          className="chevron"
+          title="View Details"
```
Comment (laurenmbeatty) [inline src/components/ListItem.js]:
```suggestion
          title={`View ${item.name} Details`}
```
If you want to have the item name...not a big deal if you don't.

**Review by laurenmbeatty — APPROVED** [review]
This looks amazing. Keep playing with it if you want but 👍 from me!

**Pull request merged**